graylog2-server/src/main/java/org/graylog2/rest/resources/system/lookup/LookupTableResource.java

```diff
@@ -421,6 +423,30 @@ public DataAdapterApi updateAdapter(@Valid @ApiParam DataAdapterApi toUpdate) {
         return DataAdapterApi.fromDto(saved);
     }
 
+    @POST
+    @Path("adapters/validate")
+    @NoAuditEvent("Validation only")
+    @ApiOperation(value = "Validate the data adapter config")
+    public ValidationResult validateAdapter(@Valid @ApiParam DataAdapterApi toValidate) {
```

Comment:
Editing a data adapter and leaving an empty `name` does not highlight the field as not valid. The response from the server in this case is a 400 instead of 200. I guess it's because hibernate is validating the `DataAdapterApi` and it can't find a value for `name`.

I think we can live with this, as the form is in the end validated by the browser and catches that issue, but I also guess is a bit confusing when you don't know how the code underneath works.